alembic/env.py

```diff
@@ -5,13 +5,13 @@
 
 from alembic import context
 from app.database import Base
-import config as conf
+from config import DbSettings
 
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
 
-config.set_main_option('sqlalchemy.url', conf.SQLALCHEMY_DATABASE_URL)
+config.set_main_option('sqlalchemy.url', DbSettings().db_url)
```

Comment:
would be great to init DbSettings once in config file and just use it's instance thought out the app